.github/workflows/release-chocolatey.yml

```diff
@@ -0,0 +1,42 @@
+name: Release Chocolatey Package
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release'
+        required: true
+  
+jobs:
+  release:
+    name: Publish to Chocolatey Community
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set Version
+        id: release_version
+        run: |
+          if ( "${{ github.event_name }}" -eq "workflow_dispatch" ) {
+            $version = "${{ github.event.inputs.version }}"
+          }
+          else {
+            $version = "${{ github.event.release.tag_name }}" -replace "^v", ""
+          }
+          echo "Setting version to $version"
+          echo "::set-output name=version::$version"
+
+      - name: Make nuspec from the template
+        run: |
+          cd ./deploy/chocolatey
+          pwsh -File ./replace.ps1 -version ${{ steps.release_version.outputs.version }}
+        
+      
+      - name: Run Chocolatey Pack
+        run: |
+          cd ./deploy/chocolatey
+          choco pack ./asyncapi-cli.nuspec
+          choco apikey add --source "'https://push.chocolatey.org/'" --key ${{ secrets.CHOCOLATEY_API_KEY }}
+          choco push ./asyncapi.${{ steps.release_version.outputs.version }}.nupkg --source "'https://push.chocolatey.org/'"
```

Comment:
.

Reply:
please also add failure handling so we know if deploy to chocolatey fails -> https://github.com/asyncapi/cli/blob/master/.github/workflows/bump-homebrew-formula.yml#L32